<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="01HW247159" name="com.TestSuite.Customer180.MyDesk_C1_013" tests="1" failures="0" timestamp="25 Oct 2016 13:59:59 GMT" time="58.498" errors="1">
  <testcase name="MyDesk_C1_013_Method" time="58.498" classname="com.TestSuite.Customer180.MyDesk_C1_013">
    <error type="java.lang.AssertionError" message="null script failed, TalkSafe - null.jpg for more details.">
      <![CDATA[java.lang.AssertionError: null script failed, TalkSafe - null.jpg for more details.
	at org.testng.Assert.fail(Assert.java:94)
	at com.Engine.Reporter.exception(Reporter.java:107)
	at com.WebActions.WebActions.VerifyElementPresentAndClick(WebActions.java:875)
	at com.BusinessModules.MyDesk.Cases_Page.enter_the_action_to_search(Cases_Page.java:157)
	at com.TestSuite.Customer180.MyDesk_C1_013.MyDesk_C1_013_Method(MyDesk_C1_013.java:131)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- MyDesk_C1_013_Method -->
</testsuite> <!-- com.TestSuite.Customer180.MyDesk_C1_013 -->
